project-reports/2023/2023-Q2-Hyperledger-Aries.md

```diff
@@ -0,0 +1,211 @@
+    has deployments of [Aries Framework JavaScript] using AnonCreds objects
+    rooted in [cheqd.io] and [Cardano].
+    - BC Gov published an [AnonCreds Code With Us Opportunity] to integrate
+    [Hyperledger AnonCreds] into [Aries Cloud Agent Python]. [Indicio]
+    won that Opportunity and is working the implementation.
+- An implementation of [OpenID4VCs] (issuance) with [Aries Framework JavaScript]
+has been completed. This is the first experiment in using [OpenID4VCs] with Aries.
+The community is also considering the feasibility of issuing and presenting [ISO
+mdocs] verifiable credentials using Aries Frameworks.
+-   The effort to move off the Indy SDK continued this quarter with the
+    following results:
+    - BC Gov published an [Askar Code With Us Opportunity] to create a
+    conversion script from Indy SDK to [Aries Askar] storage. [Indicio]
+    won that Opportunity, completed that implementation, and the
+    result has been used to convert a production system.
+    - [Animo Solutions] won a similar [AFJ Askar Code With Us Opportunity] and
+    has demonstrated a deployment of [Aries Bifold] using Aries Askar storage.
+-   After being awarded a BC Gov [Mobile Verifier Code With Us] opportunity, [DSR]
+    implemented in the Aries Bifold wallet the ability for one wallet to request
+    a presentation from another wallet, and for two wallets to establish a connection
+    and communicate (messaging and presentation requests).
+-   Discussions continue on Aries Interop Profile (AIP) v3.0, which extend
+    AIP 2.0 to be based on DIDComm V2, and a number of Aries community
+    members contributed to a DIDComm V2.0 Connect-a-thon at IIW.
+-   Continued good progress was made in the community on the use of [Overlays
+    Capture Architecture](https://oca.colossi.network/) (OCA) to power the
+    on-screen display of credentials, as mentioned in previous reports. An Aries
+    RFC has been proposed and gone through a couple of iterations, with good
+    feedback -- [OCA for Aries RFC
+    PR](https://github.com/hyperledger/aries-rfcs/pull/755), [OCA for Aries RFC
+    (proposed)](https://github.com/swcurran/aries-rfcs/tree/oca4aries/features/0755-oca-for-aries)
+    and [OCA for Aries Style Guide
+    (proposed)](https://github.com/swcurran/aries-rfcs/tree/oca4aries/features/0756-oca-for-aries-style-guide).
+    -   OCA has been implemented in [Aries Bifold]. However, we have had a
+    challenge in figuring out the best way to distribute OCA Bundles. As a short
+    cut around the problem the community has created a GitHub [OCA Bundles
+    repository], with a governance model allowing anyone to submit a (valid) OCA
+    Bundle for their credential type. The repo is currently in the BC Gov
+    GitHub organization, but may be moved as OCA Bundles become more prevalent in the
+    community. A tool, the [OCA Explorer] has been built to make it easy to see
+    and evolve OCA Bundles implementations. Check it out!
```

Comment:
No big deal this time, but usually this detailed information goes into the "Overall Activity in the Past Quarter" section.